src/controller/healthController.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+export const getHealthController = async (req, res) => {
+  try {
+    res.status(200).json({ data: "Health connected to database" });
```

Comment:
getHealthController is not checking de database connection, it's always returning with status 200

Reply:
I think it has been fixed now